app/controllers/reservations_controller.rb

```diff
@@ -256,7 +263,9 @@ def reservation_params
       :actual_start_meridian,
     )
 
-    reservation_params.merge!(reservation_start_as_params) if fixed_start_time?
+    # If the start time is locked (only after purchase, don't allow the params
```

Comment:
Can you clarify this comment? Should this be something like "Prevent override of start time params after purchase if start time is locked?"

Also, under what circumstances is the start time locked? If it's inside the lock window? Should we just say that?

Reply:
It could be within the lock window, or after the reservation has started. Basically it's any of the times where you can extend the reservation, but not move the start time. But I can work on cleaning up the wording.